tests/ui/auxiliary/macro_rules.rs

```diff
@@ -57,3 +57,10 @@ macro_rules! bad_transmute {
         std::mem::transmute($e)
     };
 }
+
+#[macro_export]
+macro_rules! empty_struct {
+    ($i:ident) => {
```

Comment:
Is there a reason this takes an ident? It seems unused in the macro

Reply:
That was an oversight. Moreover, it doesn't require using an aux-build here. Fixed.